auth/src/android/auth_android.cc

```diff
@@ -309,6 +309,14 @@ void Auth::DestroyPlatformAuth(AuthData* auth_data) {
   }
 }
 
+void LogHeartbeat(Auth* auth) {
+  // Calling the native getter is sufficient to cause a Heartbeat to be logged.
+  JNIEnv* env = Env(auth->auth_data);
+  jobject platform_app = auth->app()->GetPlatformApp();
```

Comment:
Remember to call env->DeleteLocalRef(platform_app)

Reply:
Done.